lib/google_scraping/dashboard/dashboard.ex

```diff
@@ -0,0 +1,61 @@
+defmodule GoogleScraping.Dashboard do
+  @moduledoc """
+  The Dashboard context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias GoogleScraping.Dashboard.Queries.KeywordQueries
+  alias GoogleScraping.Dashboard.Schemas.Keyword
+  alias GoogleScraping.Repo
+
+  @doc """
+  Returns the list of keywords associated with the given user.
+
+  ## Examples
+
+      iex> list_keywords(user_id)
+      [%Keyword{}, ...]
+
+  """
+  def list_keywords(user_id) do
+    Repo.all(KeywordQueries.user_keywords_query(user_id))
+  end
+
+  @doc """
+  Creates a keyword.
+
+  ## Examples
+
+      iex> create_keyword(%{field: value})
+      {:ok, %Keyword{}}
+
+      iex> create_keyword(%{field: bad_value})
+      {:error, %Ecto.Changeset{}}
+
+  """
+  def create_keyword(attrs \\ %{}) do
+    %Keyword{}
+    |> Keyword.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  @doc """
+  Creates keywords list.
+
+  TODO start background jobs for each keyword.
```

Comment:
We mostly follow todo(`TODO:`) with a comment like `# TODO: your comment`. Many IDEs have special support for this todo pattern

Reply:
In the first version, I had the comment as you suggested, but with default settings, linter doesn't allow to add such comments. Are you disabling this rule in client projects?

<img width="859" alt="image" src="https://user-images.githubusercontent.com/475367/170924051-63b34cc3-44f3-48ec-be33-85f2bcd698a5.png">

Reply:
In our current client project(Worpt) we also have those, but those doesn't cause us to fail the CI. If it does not falls CI, we can keep it.

![Screen Shot 2022-05-30 at 3 11 37 PM](https://user-images.githubusercontent.com/6076762/170947928-64e8a1f2-7295-45d5-9399-a2c5041f141c.png)

Reply:
Thanks, for some reason I thought that it will fail CI